# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../transcode/avifile.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- transcode-1.0.5/import/af6_decore.cpp.orig	2005-07-12 08:49:27.000000000 +0200
+++ transcode-1.0.5/import/af6_decore.cpp	2008-01-08 21:47:16.000000000 +0100
@@ -43,6 +43,8 @@
 #include <avifile-0.7/version.h>
 #include <avifile-0.7/renderer.h>
 #include <avifile-0.7/avm_creators.h>
+#include <avifile-0.7/videodecoder.h>
+#include <avifile-0.7/audiodecoder.h>
 #elif HAVE_AVIFILE_INCLUDES == 0
 #include <avifile/avm_fourcc.h>
 #include <avifile/avifile.h>
